src/run.rs

```diff
@@ -58,10 +59,32 @@ pub fn run(
                             context.push_msg(e.to_string());
                         }
                     }
+                    Event::Key(Key::Char('`')) => {
```

Comment:
This should be delegated to its own command like `goto_bookmark` or something.
Then it can be mapped by users to any key.
You can take some inspiration from `open_with` and how joshuto currently handles multi-key command mappings.